RecoTracker/IterativeTracking/python/PixelPairStep_cff.py

```diff
@@ -95,6 +134,13 @@
     #    cms.PSet( refToPSet_ = cms.string('pixelPairStepTrajectoryFilterShape'))
     ),
 )
+eras.trackingPhase2PU140.toReplaceWith(pixelPairStepTrajectoryFilter, TrackingTools.TrajectoryFiltering.TrajectoryFilter_cff.CkfBaseTrajectoryFilter_block.clone(
+    maxLostHitsFraction = 1./10.,
+    constantValueForLostHitsFractionFilter = 0.701,
+    minPt = cms.double(0.1),
+    minimumNumberOfHits = cms.int32(3),
+    )
+)
```

Comment:
I'd suggest to customize `pixelPairStepTrajectoryFilterBase` instead (similarly to `trackingPhase1PU70` on lines 124-127). I'd expect something like
```
eras.trackingPhase2PU140.toReplaceWith(pixelPairStepTrajectoryFilterBase, _pixelPairStepTrajectoryFilterBase.clone(
    maxLostHitsFraction = 1./10.,
    constantValueForLostHitsFractionFilter = 0.701,
))
```
to be enough.